### ISSUE TITLE
larger batchsize, bad results?

### ISSUE THREAD

**Issue by jeannotes** [open-mmlab/mmdetection#6160]
when I train my own dataset using retinanet with batchsize-16, the map gets about 85%,
and I only enlarge the batchsize to 36, the map is 0
why does this happens, I train both using 2 gpus, I can confirm that I only change batchsize.

**Comment by BIGWangYuDong** [comment]
https://github.com/open-mmlab/mmdetection/blob/13880ae5fdfa2521060e377b0908148cc291eac0/docs/1_exist_data_model.md#train-predefined-models-on-standard-datasets

**Comment by jeannotes** [comment]
@BIGWangYuDong sorry，I didn't find any references or any solutions

**Comment by ZwwWayne** [comment]
You need to also increase the learning rate linearly to match the bigger batch size.

**Comment by BIGWangYuDong** [comment]
you can try to use auto-lr during training. feel free to re-open this issue if you still meet this problem
